docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1596,6 +1600,8 @@ must restart the server for that change to take effect.
 |`fhirServer/bulkdata/validBaseUrls`|Y|Y|
 |`fhirServer/bulkdata/maxInputPerRequest`|Y|Y|
 |`fhirServer/bulkdata/validBaseUrlsDisabled`|Y|Y|
+|`fhirServer/bulkdata/cosFileMaxResources`|N|Y|
+|`fhirServer/bulkdata/cosFileMaxSize`|N|Y|
```

Comment:
should this be server wide?

Reply:
this is server wide at present.

Reply:
Right, is this something that should be configured by the tenant

Reply:
this is the same as isExportPublic config, they are all server wide at present. we have another issue to make them per tenant.

Reply:
issue #943